### PR TITLE
add securityhub-mfa-enabled-for-iam-console-access

### PR DESCRIPTION
**Pull request by jihygk**

### CODE REVIEW

File: auto_remediate/security_hub_rules.py [hunk 1]
```diff
@@ -602,6 +602,30 @@ def iam_user_unused_credentials_check(self, resource_id):
...
+        Returns:
+            boolean -- True if remediation is successful
+        """
+        try:
+            page_user = self.client_iam.get_paginator("list_users").paginate()
+            for username in page_user.search(
+                f"Users[?UserId == '{resource_id}'].UserName"
+            ):
+                self.client_iam.delete_login_profile(UserName=username)
+                self.logging.info(f"Deleted login profile for user {username}.")
```
Comment (mlevit) [inline auto_remediate/security_hub_rules.py]: Just for consistency sake can you change to: `self.logging.info(f"Deleted login profile for IAM User '{username}'.")`

File: auto_remediate/security_hub_rules.py [hunk 2]
```diff
@@ -602,6 +602,30 @@ def iam_user_unused_credentials_check(self, resource_id):
...
+                return True
+        except:
+            self.logging.error(
```
Comment (mlevit) [inline auto_remediate/security_hub_rules.py:623]: Change to: `self.logging.error(f"Could not delete login profile for IAM User '{resource_id}'.")`

**Review by mlevit — CHANGES_REQUESTED** [review]
Couple of small changes plus you forgot to update the README

File: auto_remediate/security_hub_rules.py (hunk 1 above)
Comment (jihygk) [reply auto_remediate/security_hub_rules.py]: 8335333a412075e2efc6eb2bdc9621eafa096771

File: auto_remediate/security_hub_rules.py (hunk 2 above)
Comment (jihygk) [reply auto_remediate/security_hub_rules.py:623]: 8335333a412075e2efc6eb2bdc9621eafa096771

**Review by mlevit — APPROVED** [review]

**Pull request merged**